build.sbt

```diff
@@ -11,6 +11,7 @@ lazy val gcsfilesystem = (project in file("filesystems/gcs"))
 
 lazy val database = (project in file("database"))
   .settings(databaseSettings:_*)
+  .dependsOn(core % "test->test")
```

Comment:
I thought the same thing. The only reason is to avoid duplicating Tags (eg PostMVP that can apply to any of the subprojects)